src/components/html/EcAccount.html

```diff
@@ -0,0 +1,30 @@
+<div class="ec-account">
+  <div v-if="isUserIdentified" class="ec-account-logged" key="account">
+    <div class="py-3">
+      <h2>{{ i18n('HelloAgain') + ' ' + nickname }}</h2>
+      <small>
+        {{ i18n('IsNotYou') + '?'}}
+        <a href="#" @click.prevent="logout" class="ec-account-logout">
+          {{ i18n('Logout') }}
+        </a>
+      </small>
+    </div>
+    <b-tabs content-class="mt-3">
+      <b-tab :title="i18n('Registration')" active>
+        <ec-account-form :customer.sync="localCustomer"/>
+      </b-tab>
+      <b-tab :title="i18n('Addresses')">
+        <ec-addresses :customer.sync="localCustomer" v-if="localCustomer.addresses.length"></ec-addresses>
+        <ec-address-form :customer.sync="localCustomer" v-else></ec-address-form>
```

Comment:
o `EcAddresses` por si só já mostra o formulário se o customer não tiver endereços...

```suggestion
        <ec-addresses :customer.sync="localCustomer"/>
```